src/server/users/controllers.js

```diff
@@ -1,9 +1,58 @@
-
+import {KeyUtils, ENTITY, STATE} from '../key-utils';
+import {S3Connector} from '../aws-s3';
+import db from '../database';
+import {APIError} from '../ErrorHandler';
 export default {
   get() {
 
   },
-  post() {
-
+  post: (req, res, cb) => {
+    const currentTime = new Date();
+    const timeHash = KeyUtils.genTimeHash(currentTime);
+    const key = `${ENTITY.USER}-${timeHash}`;
+    const idxKey = KeyUtils.getPrefix(ENTITY.USER, STATE.ALIVE, timeHash);
+    const user = {
+      key: key,
+      name: req.body.name,
+      email: req.body.email,
```

Comment:
I think we don't have to store user's account type or other Facebook information yet.
So I just store email & password without regarding to account type.